dashboard/app/models/user.rb

```diff
@@ -683,6 +685,15 @@ def self.find_by_hashed_email(hashed_email)
     migrated_user || User.find_by(hashed_email: hashed_email)
   end
 
+  def primary_contact_info
+    if Policies::Lti.lti?(self)
+      ao_with_email = authentication_options.select {|ao| ao.email.present?}
+      return ao_with_email.first unless ao_with_email.empty?
```

Comment:
Also I'm realizing here that we should return `primary_contact_info` even for LTI users, if it exists, otherwise return `authentication_options` which is needed during the LTI based signup flow